@swmitra I think it might be some sort of edge case, but I have been seeing things like

``` js
ErrorNotification.js:122 [Extension] failed to load /Users/Nykanen/Library/Application Support/Brackets/extensions/user/zaggino.brackets-eslint - Error: Preference brackets-eslint.gutterMarks was redefinedwindow.console.error @ ErrorNotification.js:122
/utils/ExtensionLoader.js:231 Error: Preference brackets-eslint.gutterMarks was redefined
    at PreferencesSystem._.extend.definePreference (/preferences/PreferencesBase.js:1314)
    at Object.PrefixedPreferencesSystem.definePreference (/preferences/PreferencesBase.js:1081)
    at Object.<anonymous> (main.js:18)
    at Object.context.execCb (require.js:1658)
    at Object.Module.check (require.js:869)
    at Object.<anonymous> (require.js:1121)
    at require.js:132
    at require.js:1164
    at each (require.js:57)
    at Object.Module.emit (require.js:1163)
    at Object.Module.check (require.js:925)
    at Object.Module.enable (require.js:1151)
    at Object.Module.init (require.js:782)
    at callGetModule (require.js:1178)
    at Object.context.completeLoad (require.js:1552)
    at HTMLScriptElement.context.onScriptLoad (require.js:1679)
```

when trying to update to the latest, non-packaged version.

![image](https://user-images.githubusercontent.com/7641760/31113782-83359f70-a823-11e7-8790-3aef7b6ee4c1.png)

--- a/README.md
+++ b/README.md
@@ -1,6 +1,6 @@
 # ESLint [![Build Status](https://travis-ci.org/brackets-userland/brackets-eslint.svg?branch=master)](https://travis-ci.org/brackets-userland/brackets-eslint)
 
-> **Heads up!** Beginning with version 1.11, Brackets now includes a prepackaged version of ESLint. If you want to get the possible benefits of `brackets-eslint` such as quicker release cycle, you might have to uninstall the default extension first via Extension Manager -> Installed -> ESLint.
+> **Heads up!** Beginning with version 1.11, Brackets now includes a prepackaged version of ESLint. If you want to get the possible benefits of the standalone `brackets-eslint` such as a faster release cycle, you might have to uninstall the default extension first via Extension Manager -> Installed -> ESLint (and vice versa if you want to use the prepackaged one).
 
 Brackets extension which provides file linting with ESLint.
 

